### PR TITLE
update(data/zh): update helm installation

### PR DESCRIPTION
**Pull request by ysicing**
**What type of PR is this?**

> /kind content

**Any specific area of the project related to this PR?**

> /area documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

The current installation may not work, need to update helm repo

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/zh/docs/installation.md [hunk 1]
```diff
@@ -33,12 +33,14 @@ weight: 2
...
-Helm是在Kubernetes上安装Falco的首选方法之一。[Falco Helm chart](https://github.com/helm/charts/tree/master/stable/falco)提供了丰富的[可配值](https://github.com/helm/charts/tree/master/stable/falco#configuration)，以使用不同的配置启动Falco。
+Helm是在Kubernetes上安装Falco的首选方法之一。[Falco Helm chart](https://github.com/falcosecurity/charts)提供了丰富的[可配值](https://github.com/falcosecurity/charts/tree/master/falco#configuration)，以使用不同的配置启动Falco。
 
 在部署Helm的集群上，以默认配置部署Falco，请运行：
 
 ```shell
-helm install --name falco stable/falco
+helm repo add falcosecurity https://falcosecurity.github.io/charts
+helm repo update
+helm install --name falco falcosecurity/falco
```
Comment (leogr) [inline content/zh/docs/installation.md:43]:
```suggestion
helm install falco falcosecurity/falco
```
Helm3 won't work with `--name`.

**Review by leogr — COMMENTED** [review]
Although the `zh` translation is not really up-to-date, it still worth fixing those commands.
So thank you!

I have just left a comment, otherwise, it seems good to me

**Comment by leogr** [comment]
Thank you for adding https://github.com/falcosecurity/falco-website/pull/308/commits/ce9ba6d0a94ce2439d0248d28f0c6244f9b19505 but I have noticed the [commit sign-off](https://github.com/falcosecurity/.github/blob/master/CONTRIBUTING.md#developer-certificate-of-origin) is missing, could you add that, please?

**Comment by ysicing** [comment]
@leogr  ok, added

**Review by leogr — APPROVED** [review]
Thank you!

**Pull request merged**